.pre-commit-config.yaml

```
@@ -47,4 +47,4 @@ repos:
         language: system
         types: [python]
         pass_filenames: false # Run full test suite
-        stages: [pre-commit] # Only run on commit
+        stages: [pre-commit] # Only run on commit
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)